config/datatypes_conf.xml.sample

```diff
@@ -429,6 +429,7 @@
     <datatype extension="hmm3" type="galaxy.datatypes.msa:Hmmer3" display_in_upload="true" />
     <datatype extension="stockholm" type="galaxy.datatypes.msa:Stockholm_1_0" display_in_upload="True" />
     <datatype extension="xmfa" type="galaxy.datatypes.msa:MauveXmfa" display_in_upload="True" />
+    <datatype extension="CEL" type="galaxy.datatypes.binary:Cel" display_in_upload="True" />
```

Comment:
I think we recommend the extension to be lowercase, see also: https://travis-ci.org/galaxyproject/galaxy/jobs/168975520#L1056